### DISCUSSION TITLE
Does ui.image support `svg`s?

### DISCUSSION THREAD

**Discussion by johnlarkin1** [zauberzeug/nicegui#1504] [Q&A] [answered]
### Question

I have a URL that renders fine that is a svg. However, when I use `ui.image(path_to_svg_url)` it does not render successfully. 

I'm afraid I know the answer, but I just wanted to see if for some reason this was not the expected behavior.

**Comment by falkoschindler** [comment, marked as answer]
Just try it out with some dummy SVG:
```py
ui.image('https://www.svgrepo.com/show/4733/samples.svg')
```
It works.

**Reply by johnlarkin1** [reply]
Cool... Yeah I wonder if it's some internal network issue then with why it isn't loading. The documentation page and examples also weren't loading so it's probably a networking / security conflict. 

No worries thanks for the quick replies!
